content/continued-learning.qmd

```diff
@@ -31,7 +31,7 @@ For continued learning, please consider the following external resources.
 
 ### Coordinate systems
 
-* [ESRI: Coordinate Systems: What's the
```

Comment:
Huh, I guess they have branded as `Esri` over `ESRI`! ESRI == "Environmental Systems Research Institute" and I've always capitalized it.

Reply:
I've seen it branded as `esri` too. I got a bit confused :laughing: